app/models/form_profile.rb

```diff
@@ -105,7 +105,7 @@ class FormProfile
     '40-10007'       => ::FormProfiles::VA4010007,
     '21P-527EZ'      => ::FormProfiles::VA21p527ez,
     '22-0993'        => ::FormProfiles::VA0993,
-    'COMPLAINT-TOOL' => ::FormProfiles::ComplaintTool
+    'FEEDBACK-TOOL' => ::FormProfiles::FeedbackTool
```

Comment:
nit: this breaks the fancy vertical alignment, which if it were up to me I'd totally remove since it's a PITA to maintain :)